draft-ietf-httpbis-http2bis.xml

```diff
@@ -3732,29 +3736,29 @@
           <xref target="ENHANCE_YOUR_CALM" format="none">ENHANCE_YOUR_CALM</xref>.
         </t>
         <section anchor="MaxHeaderBlock">
-          <name>Limits on Header Block Size</name>
+          <name>Limits on Field Block Size</name>
           <t>
-            A large <xref target="HeaderBlock">header block</xref> can cause an implementation to
-            commit a large amount of state.  Header fields that are critical for routing can appear
-            toward the end of a header block, which prevents streaming of header fields to their
+            A large <xref target="HeaderBlock">field block</xref> can cause an implementation to
+            commit a large amount of state.  Field lines that are critical for routing can appear
+            toward the end of a field block, which prevents streaming of fields to their
             ultimate destination.  This ordering and other reasons, such as ensuring cache
-            correctness, mean that an endpoint might need to buffer the entire header block.  Since
-            there is no hard limit to the size of a header block, some endpoints could be forced to
-            commit a large amount of available memory for header fields.
+            correctness, mean that an endpoint might need to buffer the entire field block.  Since
+            there is no hard limit to the size of a field block, some endpoints could be forced to
+            commit a large amount of available memory for field blocks.
           </t>
           <t>
             An endpoint can use the <xref target="SETTINGS_MAX_HEADER_LIST_SIZE" format="none">SETTINGS_MAX_HEADER_LIST_SIZE</xref> to advise peers of
-            limits that might apply on the size of header blocks.  This setting is only advisory, so
-            endpoints MAY choose to send header blocks that exceed this limit and risk having the
+            limits that might apply on the size of field blocks.  This setting is only advisory, so
```

Comment:
The setting applies to the size of a decompressed header list, which implicitly limits the size of the blocks (it is compression, after all).  Does that need to be called out more explicitly?